migration/migrating-4-4/migrating-openshift-4-to-4.adoc

```diff
@@ -0,0 +1,91 @@
+[id="migrating-openshift-4-to-4"]
+= Migrating {product-title} 4.1 to 4.2
+include::modules/common-attributes.adoc[]
+:context: ocp-4_1-4_2
+:ocp-4_1-4_2:
+
+toc::[]
+
+You can migrate application workloads from {product-title} 4.1 to 4.2 with the Cluster Application Migration (CAM) tool. The CAM tool enables you to control the migration and to minimize application downtime.
+
+The CAM tool's web console and API, based on Kubernetes custom resources, enable you to migrate stateful application workloads at the granularity of a namespace.
+
+You can migrate data to a different storage class, for example, from Red Hat Gluster Storage or NFS storage to Red Hat Ceph Storage.
+
+.Prerequisites
+
+* The source cluster must be {product-title} 4.1.
+* The target cluster must be {product-title} 4.2.
+* You must have xref:../../cli_reference/openshift_cli/administrator-cli-commands.html#policy[`cluster-admin` privileges] on all clusters.
+* You must have `podman` installed.
+* You must have a replication repository that supports the S3 API and is accessible to the source and target clusters.
+* If your application uses images from the `openshift` namespace, the required versions of the images must be present on the target cluster. If not, you must xref:../../openshift_images/image-streams-manage.html#images-imagestreams-update-tag_image-streams-managing[update the `imagestreamtags` references] to use an available version that is compatible with your application.
++
+[NOTE]
+====
+If the `imagestreamtags` cannot be updated, you can manually upload equivalent images to the application namespaces and update the applications to reference them.
+
+The following `imagestreamtags` have been _removed_ from {product-title} 4.2:
+
+* `dotnet:1.0`, `dotnet:1.1`, `dotnet:2.0`
+* `dotnet-runtime:2.0`
+* `mariadb:10.1`
+* `mongodb:2.4`, `mongodb:2.6`
+* `mysql:5.5`, `mysql:5.6`
+* `nginx:1.8`
+* `nodejs:0.10`, `nodejs:4`, `nodejs:6`
+* `perl:5.16`, `perl:5.20`
+* `php:5.5`, `php:5.6`
+* `postgresql:9.2`, `postgresql:9.4`, `postgresql:9.5`
+* `python:3.3`, `python:3.4`
+* `ruby:2.0`, `ruby:2.2`
+====
+
+include::modules/migration-understanding-cam.adoc[leveloffset=+1]
+
+== Deploying the Cluster Application Migration tool
+
+Deploying the CAM tool involves the following steps:
+
+. Installing the CAM Operator on all clusters
++
+The CAM Operator installs the Migration controller, CAM web console, Restic, and Velero on the target cluster and Restic and Velero on the source cluster(s).
+
+. Configuring cross-origin resource sharing on the source cluster(s) to enable communication with the CAM tool
+
+:ocp-4_1-4_2!:
+
+:context: sourcecluster
```

Comment:
Same thing here about changing the context not being something we should probably be doing.